core/src/main/java/brooklyn/entity/rebind/RebindContextLookupContext.java

```diff
@@ -60,6 +66,9 @@ public RebindContextLookupContext(ManagementContext managementContext, RebindCon
     @Override public Entity lookupEntity(String id) {
         Entity result = rebindContext.getEntity(id);
         if (result == null) {
+            result = managementContext.lookup(id, Entity.class);
```

Comment:
yes - i didn't want to break compatibility, but now i see that i did, and that the only usages are during rebind.  have removed that field and added javadoc that this class looks in both (which helps justify its existence besides the rebind context).